sdk/batch/Azure.ResourceManager.Batch/tests/TestCase/BatchAccountCertificateCollectionTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Batch.Models;
+using Azure.ResourceManager.Batch.Tests.Helpers;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Batch.Tests.TestCase
+{
+    public class BatchAccountCertificateCollectionTests : BatchManagementTestBase
+    {
+        public BatchAccountCertificateCollectionTests(bool isAsync)
+            : base(isAsync)//, RecordedTestMode.Record)
+        {
+        }
+
+        private async Task<BatchAccountCertificateCollection> GetAccountCollectionAsync()
+        {
+            var container = (await CreateResourceGroupAsync()).GetBatchAccounts();
+            var input = ResourceDataHelper.GetBatchAccountData();
+            var lro = await container.CreateOrUpdateAsync(WaitUntil.Completed, Recording.GenerateAssetName("testaccount"), input);
+            var account = lro.Value;
+            return account.GetBatchAccountCertificates();
+        }
+
+        [TestCase]
+        public async Task CertificateCollectionApiTests()
+        {
+            //1.CreateOrUpdate
+            var container = await GetAccountCollectionAsync();
+            var name = "sha1-cff2ab63c8c955aaf71989efa641b906558d9fb7";
+            var input = ResourceDataHelper.GetBatchAccountCertificateData();
+            var lro = await container.CreateOrUpdateAsync(WaitUntil.Completed, name, input);
+            BatchAccountCertificateResource certificate1 = lro.Value;
+            Assert.AreEqual(name, certificate1.Data.Name);
+            //2.Get
+            BatchAccountCertificateResource certificate2 = await container.GetAsync(name);
+            ResourceDataHelper.AssertCertificate(certificate1.Data, certificate2.Data);
+            //3.GetAll
+            _ = await container.CreateOrUpdateAsync(WaitUntil.Completed, name, input);
```

Comment:
to test GetAll properly, since here we already created an instance above, please change a name here, and assert the count >= 2

Reply:
Because the name of The BatchAccountCertificate is a definite string type, changing it will cause some unknown errors, so it has not been modified